### PR TITLE
feat: add utils tests

### PR DESCRIPTION
**Pull request by henriettemoe**

### CODE REVIEW

File: src/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,255 @@
+import { filterOutVariant, filterWord, parseWords } from "./utils";
+
+describe("Vocabulary drill utils", () => {
+  describe(filterWord.name, () => {
+    it("should return empty when epmty", () => {
```
Comment (boyum) [inline src/utils.test.ts:5]:
```suggestion
    it("should return empty when empty", () => {
```

File: src/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+    it("should parse one word correct (dragText)", () => {
+      const words = "fire,ild";
+
+      const expected = "fire *ild*\n";
+      const actual = parseWords(words, "dragText");
+
+      expect(actual).toBe(expected);
+    });
+
+    it("should parse one word correct with variant (fillIn)", () => {
```
Comment (boyum) [inline src/utils.test.ts:165]: Since there are so many tests that are tagged with fillIn or dragText, we _could_ use `describe`, but this works ✨

**Review by boyum — APPROVED** [review]

**Comment by boyum** [comment]
We need to run these tests with GitHub actions

**Comment by henriettemoe** [comment]
Added new commit @boyum

**Review by boyum — APPROVED** [review]
Nice! 🥳

**Pull request merged**